.github/workflows/ci.yml

```diff
@@ -216,11 +220,10 @@ jobs:
       - name: start abq work
         # abq work _should_ fail because we have failing tests, but we want to ensure it only has a single line of output
         run: |
-          set +e
-          echo "run worker"
-          abq work -n cpu-cores --run-id "$GITHUB_RUN_ID-$GITHUB_RUN_ATTEMPT-${{matrix.ruby}}-${{matrix.gemfile}}" 2>&1 |
-            bin/test-worker-output.rb
+          bin/test-worker-output.rb success
+          bin/test-worker-output.rb failure
```

Comment:
I'm confused about what's going on here. Why are we calling both

```
bin/test-worker-output.rb success
bin/test-worker-output.rb failure
```

Is there a test that's dynamic that is passing or failing based on the same input?

Reply:
should be cleared up in #48